index.html

```diff
@@ -116,7 +121,7 @@ <h2>
               Single node Kubernetes done right
             </h2>
             <p>
-              The smallest, fastest fully-conformant Kubernetes that tracks upstream releases and makes clustering trivial. Use MicroK8s for offline development, prototyping, testing. Use it on a VM as a small, cheap, reliable k8s for CI/CD. Makes a great kubernetes for appliances. Develop IoT apps for k8s and deploy them to MicroK8s on your boxes.
+              The smallest, fastest fully-conformant Kubernetes that tracks upstream releases and makes clustering trivial. MicroK8s is great for offline development, prototyping, and testing. Use it on a VM as a small, cheap, reliable k8s for CI/CD. The best kubernetes for appliances. Develop IoT apps for k8s and deploy them to MicroK8s on your Linux boxes.
```

Comment:
There should be another comma after "fastest" i.e. "The smallest, fastest, fully-conformant"